www/app/windows_connection.ts

```diff
@@ -26,6 +26,9 @@ export class WindowsOutlineConnection implements cordova.plugins.outline.Connect
 
   constructor(public config: cordova.plugins.outline.ServerConfig, public id: string) {
     const serverName = this.config.name || this.config.host || '';
+    ipcRenderer.on(`proxy-connected-${this.id}`, (e: Event) => {
```

Comment:
I think it's worth commenting on how this event can be received, it's not obvious :-)

Reply:
Done in 23d30f0.